app/states/ohio/ohio_downloader.py

```
@@ -1 +1,35 @@
+    STATE_NAME="Ohio",
+    STATE_ABBREVIATION="OH",
+    # DATABASE_ENGINE=engine,
+    CSV_CONFIG=CSVReaderConfig(),
+
+)
+
+
+class OhioDownloader(FileDownloaderABC):
+    config: StateConfig = OHIO_CONFIGURATION
+
+    def download(self, overwrite: bool, read_from_temp: bool) -> FileDownloaderABC:
```

Ruff (0.14.7)

Boolean-typed positional argument in function definition

(FBT001)